css/@media/pointer: give examples of coarse and fine pointers (#24335)
mdn/content: files/en-us/web/css/@media/pointer/index.md

- `none`
  - : The primary input mechanism does not include a pointing device.
- `coarse`

>   - : The primary input mechanism includes a pointing device of limited accuracy, such as a touchscreen or a smart TV remote.

Reviewer: ```suggestion
  - : The primary input mechanism includes a pointing device of limited accuracy, such as a finger on a touchscreen.
```

- `fine`

>   - : The primary input mechanism includes an accurate pointing device, such as a mouse, a touchpad, or a stylus.

Reviewer: ```suggestion
  - : The primary input mechanism includes an accurate pointing device, such as a mouse.
```
